pages/agent/v3/help/_bootstrap.md

```diff
@@ -41,7 +41,7 @@ See https://buildkite.com/docs/agent/v3/hooks for more details.
 * `--pipeline value` - The slug of the pipeline that the job is a part of [`$BUILDKITE_PIPELINE_SLUG`]
 * `--pipeline-provider value` - The id of the SCM provider that the repository is hosted on [`$BUILDKITE_PIPELINE_PROVIDER`]
 * `--artifact-upload-paths value` - Paths to files to automatically upload at the end of a job [`$BUILDKITE_ARTIFACT_PATHS`]
-* `--artifact-upload-destination value` - A custom location to upload artifact paths to (i.e. s3://my-custom-bucket) [`$BUILDKITE_ARTIFACT_UPLOAD_DESTINATION`]
+* `--artifact-upload-destination value` - A custom location to upload artifact paths to (e.g. s3://my-custom-bucket) [`$BUILDKITE_ARTIFACT_UPLOAD_DESTINATION`]
```

Comment:
This is going to conflict with #1168, which is where I originally spotted the i.e. vs e.g. thing.

Reply:
These are all generated from the agent help as well, so need updating on that side too.